examples/PolyOseen.cc

```diff
@@ -0,0 +1,1435 @@
+                    FullMatrix<double> M12(current_dofs_per_cell,
+                                           neigh_dofs_per_cell);
+                    FullMatrix<double> M21(neigh_dofs_per_cell,
+                                           current_dofs_per_cell);
+                    FullMatrix<double> M22(neigh_dofs_per_cell,
+                                           neigh_dofs_per_cell);
+                    M11 = 0.;
+                    M12 = 0.;
+                    M21 = 0.;
+                    M22 = 0.;
+
+                    const auto &normals = fe_faces0.get_normal_vectors();
+
+                    std::vector<Tensor<2, dim>> aver_grad_phi_v0(
+                      current_dofs_per_cell);
+                    std::vector<Tensor<1, dim>> jump_phi_v0(
+                      current_dofs_per_cell);
+                    std::vector<double> aver_phi_p0(current_dofs_per_cell);
+                    std::vector<double> jump_phi_p0(current_dofs_per_cell);
+                    std::vector<Tensor<1, dim>> downwind_phi_v0(
+                      current_dofs_per_cell);
+
+                    std::vector<Tensor<2, dim>> aver_grad_phi_v1(
+                      neigh_dofs_per_cell);
+                    std::vector<Tensor<1, dim>> jump_phi_v1(
+                      neigh_dofs_per_cell);
+                    std::vector<double> aver_phi_p1(neigh_dofs_per_cell);
+                    std::vector<double> jump_phi_p1(neigh_dofs_per_cell);
+                    std::vector<Tensor<1, dim>> downwind_phi_v1(
+                      neigh_dofs_per_cell);
+
+                    std::vector<Tensor<1, dim>> beta(
+                      fe_faces0.n_quadrature_points);
+                    beta_function->value_list(fe_faces0.get_quadrature_points(),
+                                              beta);
+
+                    double beta_max = 0;
+                    for (unsigned int q_index = 0;
+                         q_index < fe_faces0.n_quadrature_points;
+                         ++q_index)
+                      {
+                        double beta_max0 = beta[q_index].norm();
+                        if (beta_max0 > beta_max)
+                          beta_max = beta_max0;
+                      }
+
+                    unsigned int deg_v_current =
+                      polytope->get_fe().get_sub_fe(0, 1).degree;
+                    unsigned int deg_v_neigh =
+                      neigh_polytope->get_fe().get_sub_fe(0, 1).degree;
+                    double tau_current = (viscosity_nu) * (deg_v_current + 1) *
+                                         (deg_v_current + dim) /
+                                         std::fabs(polytope->diameter());
+                    double tau_neigh = (viscosity_nu) * (deg_v_neigh + 1) *
+                                       (deg_v_neigh + dim) /
+                                       std::fabs(neigh_polytope->diameter());
+                    double sigma_v = 40.0 * std::max(tau_current, tau_neigh);
```

Comment:
Could you define this constant as a member of the class? I would prefer not to have it hard-coded in the middle of the code.